code/modules/jobs/job_types/assistant.dm

```diff
@@ -21,6 +21,15 @@ Assistant
 	departments = DEPARTMENT_SERVICE
 	rpg_title = "Lout"
 
+	mail_goodies = list(
```

Comment:
I feel like adding a list to every single job datums is not a good idea. Instead the subsystem should hold the lists and you would filter the results based on the job.